### PR TITLE
🙈 Add bigdatarpg.com 🐼

### PR DESCRIPTION
**Pull request by BigDataRPG**
This is my personal website which I attach link on the right above the social media icon.

### CODE REVIEW

**Comment by BigDataRPG** [comment]
image

This is the icon location in the website that link to วงแหวนเว็บ

File: index.html [hunk 1]
```diff
@@ -256,6 +256,9 @@ <h1>
         <li data-lang="th" id="mesodiar.com" data-owner="mesodiar" data-feed="https://medium.com/feed/mils-blog">
           <a href="https://mesodiar.com">mesodiar.com</a>
         </li>
+        <li data-lang="th" id="bigdatarpg.com/" data-owner="bigdatarpg" data-feed="https://bigdatarpg.com/">
```
Comment (dtinth) [inline index.html:259]:
```suggestion
        <li data-lang="th" id="bigdatarpg.com" data-owner="bigdatarpg" data-feed="https://bigdatarpg.com/feed/">
```

**Review by dtinth — CHANGES_REQUESTED** [review]
Thanks for your interest! I’ve reviewed your site and found that the link is not indicative. Please change the icon to the webring’s logo or put in the text “วงแหวนเว็บ”.

File: index.html (hunk 1 above)
Comment (BigDataRPG) [reply index.html:259]: image

Added วงแหวนเว็บ logo already

Comment (BigDataRPG) [reply index.html]: Fixed suggestion and commited the new onealready

**Review by dtinth — APPROVED** [review]
@BigDataRPG Thanks, and welcome to the webring!

**Pull request merged**